app/main.py

```diff
@@ -2,0 +1,19 @@
+from .cafe import Cafe
+from .errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Consider handling `NotVaccinatedError` and `OutdatedVaccineError` separately to provide more specific feedback. For example, you could return different messages for unvaccinated friends and those with expired vaccines.